2023-11-28 12:23:56 ERROR Unable to initialize WebDriver
2023-11-28 12:23:56 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/SeleniumCapital/SeleniumCapital/pages/base_page.py", line 82, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/SeleniumCapital/SeleniumCapital/pages/base_page.py", line 605, in wait_for_target_url
    return Wait(self.browser, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcom-cIj6ROMf-py3.11/lib/python3.11/site-packages/selenium/webdriver/support/wait.py", line 101, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 

2023-11-28 12:24:09 ERROR Unable to initialize WebDriver
2023-11-28 12:24:09 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/SeleniumCapital/SeleniumCapital/pages/base_page.py", line 82, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/SeleniumCapital/SeleniumCapital/pages/base_page.py", line 605, in wait_for_target_url
    return Wait(self.browser, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcom-cIj6ROMf-py3.11/lib/python3.11/site-packages/selenium/webdriver/support/wait.py", line 101, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: